.gitlab-ci.yml

```diff
@@ -277,6 +299,72 @@ build-push-windows-image:
   stage: release
   dependencies:
     - sign-exe
+    - build-linux-image-arm64
```

Comment:
Please remove the `sign-exe` dependency since it is not required for the arm64 image.